ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -255,6 +255,7 @@ public class CommandBar: UIView, TokenizedControlInternal {
 
     private lazy var scrollViewConstraints: [NSLayoutConstraint] = {
         return [scrollView.contentLayoutGuide.heightAnchor.constraint(equalTo: scrollView.heightAnchor),
+                scrollView.contentLayoutGuide.trailingAnchor.constraint(equalTo: mainCommandGroupsView.trailingAnchor),
```

Comment:
For style, can this be moved to `updateMainCommandGroupsViewConstraints `

Reply:
`updateMainCommandGroupsViewConstraints` is only setting `mainCommandGroupsView` constraints. I think it makes more sense for a `scrollView` constraint to live inside `scrollViewConstraints`.